frontend/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Babble",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Collaborative 1 on 1 video calls for everyone.",
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "core-js": "^3.6.5",
+    "vue": "^3.0.0",
+    "vue-router": "^4.0.0-0"
```

Comment:
I searched online and didn't find "-0" included, not sure if it's needed or not.

Reply:
This should be fine because of the `^`. It signifies "compatible with version" which means it will use the most recent minor and patch versions while leaving the major version the same. So all versions `< 5.0.0`